packages/suite/src/middlewares/suite/analyticsMiddleware.ts

```diff
@@ -114,14 +114,16 @@ const analyticsMiddleware =
                 break;
             }
             case ROUTER.LOCATION_CHANGE:
-                analytics.report({
-                    type: EventType.RouterLocationChange,
-                    payload: {
-                        prevRouterUrl,
-                        nextRouterUrl: action.payload.url,
-                        anchor: redactTransactionIdFromAnchor(action.payload.anchor),
-                    },
-                });
+                if (state.suite.lifecycle.status === 'ready') {
```

Comment:
I'm not sure, I haven't examined it in detail, but exclude only `initial` status looks safer to me - maybe there are some relevant location changes possible in error statuses
```suggestion
                if (state.suite.lifecycle.status !== 'initial') {
```

Reply:
I am excluding `initial` and `loading`
b1aacb4700a36ea410483bf65dbc079dcc88adff